frontend/src/app/auth/auth.component.scss

```diff
@@ -4,14 +4,16 @@
 .auth {
   background-color: #ddd;
   text-align: center;
-  position: fixed;
-  width: 100vw;
-  height: 100vh;
+  margin: 0 -20px;
```

Comment:
Do you have another idea rather than using negative margin? I think that it may cause an unexpected behavior in the future.